Casks/scalar-azrepos.rb

```diff
@@ -12,6 +12,7 @@
   conflicts_with cask: 'scalar'
 
   depends_on formula: 'microsoft-git'
```

Comment:
When were we going to make this a cask? After the next release? or is that something we can do now?

Reply:
Do we want to merge in the other PR (to make `microsoft-git` a Cask) now, with 2.27 and the bad uninstall script?

Reply:
We talked offline, but the plan is to merge this now, then merge #4 immediately before shipping a new version of microsoft/git.